plugins/source/vault/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/cloudquery/plugins/vault/resources/plugin"
+
+	"github.com/cloudquery/plugin-sdk/v4/serve"
+)
+
+const sentryDSN = "https://64ac2cdbdb9cb64ed7d78053412495b8@o1396617.ingest.sentry.io/4505635677863936"
```

Comment:
Can we double-check this string is correct? I don't see a project for Vault in our Sentry, but I don't see this DSN used elsewhere either 🤔

Reply:
I think it is configured correctly as `source-vault` project e.g. 

![image](https://github.com/cloudquery/cloudquery/assets/1200192/e37fd948-f290-449b-81d2-28cff6b1a8a8)